OpenRA.Mods.Common/Widgets/Logic/Ingame/AddFactionSuffixLogic.cs

```diff
@@ -24,17 +24,29 @@ public AddFactionSuffixLogic(Widget widget, World world)
 				faction = world.LocalPlayer.Faction.InternalName;
 			var suffix = "-" + faction;
 
-			var buttonWidget = widget as ButtonWidget;
-			if (buttonWidget != null)
+			if (widget as ButtonWidget != null)
```

Comment:
This can simplify to `if (widget is ButtonWidget)` and likewise for the other checks below.

Reply:
Fixed.